plugins/Login/SessionInitializer.php

```diff
@@ -148,21 +111,6 @@ protected function doAuthenticateSession(AuthInterface $auth)
     }
 
     /**
-     * Returns a Cookie instance that manages the browser cookie used to store session
-     * information.
-     *
-     * @param bool $rememberMe Whether the authenticated session should be remembered after
-     *                         the browser is closed or not.
-     * @return Cookie
-     */
-    protected function getAuthCookie($rememberMe)
```

Comment:
This method is in theory API and now removed. Can we somehow restore that method and make it work as before? Otherwise we would need to add this to the developer changelog as breaking change.